examples/01-transient_analyses/03-mode_superposition_transient.py

```diff
@@ -0,0 +1,140 @@
+"""
+.. _ref_msup_transient:
+
+Mode superposition transient analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to postprocess a mode superposition transient result
+and visualize the outputs. It also shows how to select modes for the modal expansion.
+
+"""
+# Import the necessary modules
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+###############################################################################
+# Download the mode superposition transient result example. This example is
+# not included in DPF-Core by default to speed up the installation.
+# Downloading this example should take only a few seconds.
+#
+# Next, create the model and display the state of the result. This mode superposition transient
+# result file contains several individual results, each at a different timestamp, automatically
+# expanded on all available modes.
+
+# msup_transient_files = examples.find_msup_transient()
+msup_transient_files = {
+    "rst": r"D:\ANSYSDev\Sandbox\UnitTestDataFiles\expansion\msup\Transient\plate1\file.rst",
```

Comment:
is it meant to be hardcoded path?

Reply:
no, that is until a find a nicer example file to add to our example repo